<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keyIndicators_3</class>
 <widget class="QDialog" name="keyIndicators_3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1344</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>591</height>
    </rect>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLineEdit" name="peMinLineEditKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="PeCheckBoxKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Pe (min)                    (max)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="peMaxLineEditKey3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="PsCheckBoxKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>97</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Ps (max)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="psLineEditKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="earningsToDividendRatioCheckBoxKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Vinst/Utdel (min)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="earningsToDividendRatioLineEditKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="priceToNetEquityRatioCheckBoxKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Kurs/JEK (max)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="priceToNetEquityRatioLineEditKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="totalDebtToEquityRatioCheckBoxKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Skuld/Eget kapital (max)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="totalDebtToEquityRatioLineEditKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="currentRatioCheckBoxKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Balanslikviditet (min)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="currentRatioLineEditKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="netAssetValueToPriceRatioCheckBoxKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>450</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Substansv√§rde / Kurs (min)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="netAssetValueToPriceRatioLineEditKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="stockListscomboBoxKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="loadDataPushButtonKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Ladda data</string>
     </property>
    </widget>
    <widget class="QPushButton" name="turnOffFilterpushButtonKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Visa utan filter</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="yieldCheckBoxKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>500</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Direktavkastning (min)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="YieldLineEditKey3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>211</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>1051</width>
     <height>591</height>
    </rect>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QTableView" name="tableViewKey3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1031</width>
       <height>541</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
